src/modules/EventList/index.tsx

```diff
@@ -469,8 +449,8 @@ export const EventListUI = (props: EventListUIProps): ReactElement => {
                         </FilterBy>
                     </div>
                     <div className={classNames('flex flex-col items-center justify-center flex-1')}>
-                        {match(eventsQuery)
-                            .with({ status: 'loading' }, () => (
+                        {match(queryStatus)
```

Comment:
The query statuses were originally only using the `eventsQuery` one, but we're now firing 2 queries (recent & live) so makes sense to check both.